debian/changelog

```diff
@@ -1,3 +1,14 @@
+libfm-qt (1.1.0-0ubuntu7~ppa2) jammy; urgency=medium
+
+  * Backport to Jammy.
+  * Updated Lubuntu Developers copyright.
+  * Updated symbols file
```

Comment:
Add more verbosity

Reply:
Bear with me here, but... what else is there to be verbose about? I'm pretty sure I shouldn't mention all the steps I took to update the file, and given the horrific complexity of the file's guts, I don't want to go into detail about all the changes I made. Maybe you can suggest what extra verbosity to add?